### PR TITLE
IDs, ObjectManager, PersistentInfo

### PR DESCRIPTION
**Pull request by jbedard**
@joshcbarnes

### CODE REVIEW

File: src/main/java/com/allocadia/carbonite/CarbonRowMapper.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.allocadia.carbonite;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+
+import lombok.SneakyThrows;
+
+public class CarbonRowMapper<T> implements RowMapper<T> {
```
Comment (joshcbarnes) [inline src/main/java/com/allocadia/carbonite/CarbonRowMapper.java]: Might be nice to have consistent naming - carbonite? carbonated? carbon?

Comment (jbedard) [inline src/main/java/com/allocadia/carbonite/CarbonRowMapper.java]: Yeah, or some combo of those, but with consistent meaning.  The library is Carbonite, things modified/wrapped to work with the library are Corbonated? idk... I can change it to whatever if you have a preference...

Comment (joshcbarnes) [inline src/main/java/com/allocadia/carbonite/CarbonRowMapper.java]: I'd probably just name everything with 'Carbonite'...really I just think 'carbonated' is kind of cute <_<

File: src/main/java/com/allocadia/carbonite/CarboniteObjectManager.java [hunk 2]
```diff
@@ -13,9 +13,18 @@
     
     private Carbonite carbonite;
     
+    private PersistedObjectManager objectManager;
```
Comment (joshcbarnes) [inline src/main/java/com/allocadia/carbonite/CarboniteObjectManager.java]: Rename to PersistedObjectCache or something

File: pom.xml [hunk 3]
```diff
@@ -22,6 +22,14 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.aspectj</groupId>
```
Comment (joshcbarnes) [inline pom.xml:25]: test scope?

File: src/main/java/com/allocadia/carbonite/aspects/CarbonPersistenceAspect.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.allocadia.carbonite.aspects;
+
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.DeclareParents;
+import org.aspectj.lang.annotation.Pointcut;
+
+import com.allocadia.carbonite.PersistedObject;
+
+@Aspect
+public class CarbonPersistenceAspect extends PersistanceAspect {
```
Comment (joshcbarnes) [inline src/main/java/com/allocadia/carbonite/aspects/CarbonPersistenceAspect.java]: naming

**Pull request merged**